packages/amplication-server/src/core/entity/entity.service.ts

```diff
@@ -836,6 +836,13 @@ export class EntityService {
         );
       }
 
+      for (const relatedEntityField of relatedEntityFields) {
```

Comment:
can you please explain this change in a couple of words? how does it fix the issue?

Reply:
@yuval-hazaz  In the deleteRelatedField function, we are trying to get the Module by the entity id, and it can't find it because we have already deleted the entity module.